src/components/pages/ManagementRegime/ManagementRegime.js

```diff
@@ -352,11 +352,14 @@ const ManagementRegime = ({ isNewManagementRegime }) => {
           }
 
           if (!isAppOnline) {
-            showSyncToastError({
```

Comment:
oops, if we are offline, there will be no sync related toasts. Just show a normal toast.  And console that error.